pubsub/cli.rb

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env ruby
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "gcloud"
+
+# Shows using gcloud to interact with Pub/Sub
+class PubSubSample
```

Comment:
This doesn't have to be in a module.  But we used a `Samples` module to namespace the [Cloud Storage](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/storage/list_buckets.rb#L17) and [BigQuery](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/bigquery/shakespeare.rb#L17) samples.

Let's put this into a `Samples` module just for consistency.